config/config_sctTesting_lesionAx.json

```diff
@@ -73,5 +73,19 @@
 		"ToTensor": {},
 		"NormalizeInstance": {}
 		},
+        "transformation_testing": {
+                "Resample": {
+                        "wspace": 0.75,
+                        "hspace": 0.75
+                        },
+                "ROICrop2D": {
+                        "size": [48, 48]
+                        },
+                "RandomRotation": {
+                        "degrees": 30
+                        },
+                "ToTensor": {},
+                "NormalizeInstance": {}
+                },
```

Comment:
I guess we need to add this to all config files tho?

Reply:
Done (:

Reply:
You rock!